scripts/build_pkg.sh

```diff
@@ -39,6 +39,7 @@ fi
 DEPS_INSTALL_DIR=$BUILD_DIR/$CROSS_COMPILE/deps/usr
 FWUP_INSTALL_DIR=$BUILD_DIR/$CROSS_COMPILE/fwup-staging/usr
 PKG_CONFIG_PATH=$DEPS_INSTALL_DIR/lib/pkgconfig
+export ChocolateyInstall=$DEPS_INSTALL_DIR/chocolatey
```

Comment:
That sounds good. Also, while you're at it, could you rebase your intermediate commits together? It looks like there should just be one commit in the end for this change, but if there's an unrelated change in there that I missed, please separate that out.